canonical/ubuntu.com
# Update /release-cycle#ubuntu-kernel-release-cycle copy

## Done

- Applied minor copy update "X" -> "X11" as per [doc](https://docs.google.com/document/d/1-l3B7AoyDoETaL80UhlfEWCVKmMWPdy3zoF5LpsGcYQ/edit) (other doc comments not relevant to this pr)

## QA

- View the site locally in your web browser at: http://0.0.0.0:8001//release-cycle#ubuntu-kernel-release-cycle copy
- See that the requested change was made

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12649
